### ISSUE TITLE
[MSFT-247] Sample challenge page: Screen reader identifies Unlabelled graphics on the page.

### ISSUE THREAD

**Issue by veshu** [importtest/importest#42]
**Steps to Reproduce**
1. Go to the Sample challenge page: https://www.topcoder.com/challenges/30095347. 2. Make sure that the screen reader is switched on. 3. Click on the register button to make sure that the challenge prerequisite page is open. 4. Press the key combination 'insert + ctrl + g' to bring up the dialog with all the graphics on the page. 5. Notice that the graphics do not have appropriate labels.

**Expected Result**
The graphics on the page should have appropriate labels / alt text especially if they are informative. Decorative images should have null alt text so that they are ignored by screen readers. 

**Actual Result**
Screen reader doesn't read the graphic labels appropriately.

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 1. Perceivable
**WCAG Level**: A
**WCAG Success Criterion**: 1.1.1 Non-Text Content
**Screen Reader Used**: JAWS
**Accessibility Audit Tool used**: N/A

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=088f5f5d%2D9b1c%2D4b7e%2Dacbf%2D320c257c9a9a&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2FKameshwarid%2Fbug%2041
